libstuff/libstuff.h

```diff
@@ -36,7 +36,7 @@ extern void* SSIGNAL_NOTIFY_INTERRUPT;
 // Initialize libstuff on every thread before calling any of its functions
 void SInitialize(string threadName = "", const char* processName = 0);
 
-void SSetSignalHandlerDieFunc(function<void()>&& func);
+void SSetSignalHandlerDieFunc(function<string()>&& func);
```

Comment:
Maybe add some doc to (briefly) explain what this function is, but especially why we need the `func` to return `string`?

Reply:
Done